modules/ROOT/pages/_partials/push/handling-push-notifications.adoc

```diff
@@ -54,14 +54,14 @@ Cordova::
 
 NOTE: You can only develop push applications using Ionic. Plain Cordova is not supported.
 
-. Install the plugin:
-+
 [source,bash]
 ----
-$ npm install --save @ionic-native/push
+pushObject.on('notification').subscribe(data => {
+    // Handle incoming message
+});
 ----
 
-. Review the documentation at link:https://ionicframework.com/docs/native/push[Ionic Push plugin^] for details about using the plugin. 
+Review the documentation at link:https://ionicframework.com/docs/native/push[Ionic Push plugin^] for details about using the plugin. 
```

Comment:
is this still correct, given that we are no longer asking users to install the plugin?

Reply:
Installing the push plugin is a prerequisite for ionic cordova. See [import the libraries](https://deploy-preview-648--aerogear-docs-preview.netlify.com/aerogear/latest/push-notifications/#setting-up-sdk-push) section for Cordova. 
So referring to the ionic push plugin is still in place imo.

Reply:
ah, you're right, thanks